src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx

```diff
@@ -167,6 +167,7 @@ function TransactionWorkflow({
     }
     switch (interactionType) {
       case INTERACTION_TYPES.SET_CONTROLLER:
+      case INTERACTION_TYPES.TRANSFER:
       case INTERACTION_TYPES.CREATE:
       case INTERACTION_TYPES.SET_TTL_SECONDS:
       case INTERACTION_TYPES.SET_TARGET_ID:
```

Comment:
we have `antInteractionTypes` array that we could check against `interactionType` as opposed to this switch

Reply:
![image](https://user-images.githubusercontent.com/85306700/235231701-32787f68-fe06-417f-804f-c90a8a52a37c.png)

Reply:
in this instance, you can do `pdntInteractionTypes.includes(interactionType as PDNTInteractionName)`

Reply:
cool, switch to if controls